master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,82 @@
+---
+title: IP Pool Manipulation
+---
+The 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+that Calico starts up with can be manipulated and changed.  This document
+describes how to do that using the `calicoctl` utility.
+
+## Retrieving IP Pool information
+
+The current IP Pools can be retrieved with the following commmand.
+
+```shell
+calicoctl get ipPool
+```
+
+This will return information similar to the following.
+
+```shell
+CIDR
+192.168.0.0/16
+
+```
+
+To retrieve the information about a specific IP Pool you must specify the pool
+also.  The `-o yaml` will output in a format that can be captured, manipulated,
+and then used to apply the desired changes.
+
+```shell
+calicoctl get ipPool 192.168.0.0/16 -o yaml
+```
+
+## Manipulating an existing pool
+
+### Retrieve current IP Pool config
```

Comment:
Manipulating pools is a 3 step process, so number these headings. You can keep them as trip-# if you like, but:

### 1. Retrieve current IP Pool config

### 2. Modify IP Pool Config

### 3. Apply Modified Pool